docs/4.2/admin-guide.md

```diff
@@ -1016,6 +1017,9 @@ and 3024 on the proxy. Port 3080 is used to initially fetch the credentials (SSH
 and for discovery (where is the reverse tunnel running, in this case 3024). Port 3024 is used to
 establish a connection to the Auth Server through the proxy.
 
+To enable multiplexing so only one port is used simply set the `tunnel_public_addr` and `tunnel_listen_addr` the same as the `public_addr`
```

Comment:
is used simply -> is used, simply (add comma).

Also, I don't think this is actually what enables multiplexing. Explicitly, `listen_addr` and `tunnel_listen_addr` have to be set to the same port. There is also a log message printed to do with multiplexing when Teleport is started in this mode, which we might want to highlight so people can know they're doing it right.

Reply:
I don't see the log message in 4.3 alpha.  I did confirm it works in 4.3 and 4.2.

Reply:
changed the language.

Reply:
Found the output if set to debug